src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -226,6 +227,8 @@ def launch_fluent(
     The allocated machines and core counts are queried from the scheduler environment and
     passed to Fluent.
     """
+    if version != "3d" and gpu:
```

Comment:
👍

Reply:
@seanpearsonuk  No. If we move current check to _process_invalid_args then checking current condition (`if version != "3d" and gpu:`) after execution of following 6 statements is redundant and therefore to avoid this we have added this check at first line of function.

![image](https://github.com/ansys/pyfluent/assets/106588300/add7178d-aa73-45df-ba81-ea7c7fda06c5)